features/comments/commenting.feature

```diff
@@ -176,3 +180,74 @@ Feature: Commenting
     And I should see 20 comments
     And I should see "Displaying comments 51 - 70 of 70 in total"
     And I should not see the pagination "Next" link
+
+  Scenario: Not authorized to list comments
+    Given 5 comments added by admin with an email "commenter@example.com"
+    And 3 comments added by admin with an email "admin@example.com"
+    And a show configuration of:
+      """
+        class ListCommentsForCommenterOnly < ActiveAdmin::AuthorizationAdapter
```

Comment:
This class name does not match what it does, maybe better `NeverListComments` or something like that?